src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
-                    of the first "Notification of Refusal" in respect of the original application.
-                    In any other case, a new application and a full fee must be submitted.</p>
-                <p> When modification/rectification has been completed, or additional evidence of
-                    compliance obtained, an application for a re-examination (verbally or in writing)
-                    should be made, in the case of an M1 or N1 vehicle class listed below, to attend at the test
-                    station that issued this notification and in respect of any other class or category, to
-                    the test station of your choice. To obtain a re-examination appointment please contact <strong>(0300) 200 1188</strong> </p>
-
-                <div class="exam-list">
-                    <ul id="left-exam-list">
-                        <li>Amateur Built Vehicle</li>
-                        <li>Rebuilt Vehicle</li>
-                        <li>Motor Ambulance</li>
-                        <li>Motor Caravan</li>
-                        <li>Vehicle manufactured using parts of a registered vehicle</li>
-                    </ul>
-
-                    <ul id="right-exam-list">
-                        <li>Disabled Person's Vehicle</li>
-                        <li>Hearse</li>
-                        <li>Armoured Vehicle</li>
-                        <li>Vehicle manufactured in very low volume</li>
-                    </ul>
-                </div>
+<head>
+    <title>IVA30</title>
+    <style type="text/css" media="print, screen">
+            {{~> assets/stylesheets/iva-30 this}}
+    </style>
+</head>
 
-                <div class="refusal-list">
-                    <p> Where a "Notification of Refusal" is issued, a further examination (a retest) will be carried
-                        out without further payment if: </p>
-                    <p> - the vehicle is submitted for the examination at the place where it was previously examined, and</p>
-                    <p> - the vehicle is submitted for examination before the end of the fifth day on which DVSA is accepting
-                        vehicles for inspection following the day of the original examination, and</p>
-                    <p> - in the case of any vehicle category, the application is due to a failure to provide sufficient
-                        documentary evidence before the examination to demonstrate compliance, or</p>
-                    <p> - in the case of an M1 or N1 category vehicle only, failure is due to one or more of the following
-                        items (subject to a maximum of six items)</p>
+<body>
+<div class="separate-page">
+    <!-- Header Section -->
+    <div class="page-header">
+        <div class="header-left">
+            <div class="header-serial-no">
+                <div class="header-serial-no-title">
+                    <span> <b>Serial Number: </b> <!-- Put Serial Number data here --> </span>
```

Comment:
Width of Serial Number box on original cert looks to be defined, rather than dynamic.